cmsl1t/utils/root_glob.py

```diff
@@ -64,6 +83,7 @@ def iglob(pathname):
     test_paths = [
         "data/*root",
         "data/L1Ntuple_test_3.root",
+        "d*/*root",
```

Comment:
This particular path can also be tested in our unit tests:
https://github.com/cms-l1t-offline/cms-l1t-analysis/blob/master/test/utils/test_root_glob.py

Could you please add it?

Reply:
Done